client/components/BlogPost.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import NextLink from 'next/link'
+import { useColorMode, Heading, Text, Flex, Box, Link } from '@chakra-ui/react'
+import { parseISO, format } from 'date-fns'
+
+const BlogPost = ({ title, publishedAt, summary, slug }) => {
+    const { colorMode } = useColorMode()
+    const secondaryTextColor = {
+        light: 'gray.700',
+        dark: 'gray.400'
```

Comment:
It might be a good idea to consolidate style values in a central location (e.g. `colors.js` or something) since these values may be used in multiple places